### PR TITLE
Adding Docs for Truffle Scripts

### PR DESCRIPTION
**Pull request by salazarguille**
It includes:

- Docs to detail how to execute truffle scripts.

### CODE REVIEW

File: docs/scripts.md [hunk 1]
```diff
@@ -1,26 +1,66 @@
 # Truffle Scripts
```
Comment (ark4n631) [inline docs/scripts.md:1]: shouldn't the filename would be `docs/truffle-scripts.md` just to be consistent with the other docs

**Comment by ark4n631** [comment]
Just as i commented to you in slack, i think:
image
because this project doesn't work by default on nodeJS LTS `v12.x` so you are forced to use an older nodeJS version, that should be warned before you `yarn install` and it seems hidden in a note below check truffle (when all should be already installed):

image

File: docs/scripts.md (hunk 1 above)
Comment (salazarguille) [reply docs/scripts.md]: Done

**Comment by salazarguille** [comment]
Great, I pushed the changes.

**Pull request merged**